examples/subplots_axes_and_figures/axis_labels_demo.py

```diff
@@ -0,0 +1,20 @@
+"""
+================
+Axis Labels Demo
```

Comment:
More precisely:
```suggestion
Axis Label Position
```
(also for the filename)